phlop/procs/runtimer.py

```
@@ -26,63 +24,99 @@
-        ekwargs = {}
+        ekwargs = dict(
+            shell=shell,
+            env=benv,
+            close_fds=True,
+            stdout=subprocess.DEVNULL,
+            stderr=subprocess.DEVNULL,
+        )
         if not capture_output and log_file_path:
             ekwargs.update(
                 dict(
                     stdout=open(f"{log_file_path}.stdout", "w"),
```

Ruff

Use a context manager for opening files

(SIM115)